cpu/stm32/periph/usbdev.c

```diff
@@ -513,6 +523,7 @@ static usbdev_ep_t *_usbdev_new_ep(usbdev_t *dev, usb_ep_type_t type,
             ep->dev = dev;
             ep->len = buf_len;
             usbdev->occupied += buf_len;
+            usbdev->occupied = _round_up_to_multiple_of_four(usbdev->occupied);
```

Comment:
@bergzand an alternative here would be adding

```C
    assert((buf_len & 3) == 0);
```

Would that make more sense?

Reply:
I'm fine with the current solution